bot/engine/src/main/kotlin/engine/nlp/Nlp.kt

```diff
@@ -258,7 +259,8 @@ internal class Nlp : NlpController {
                 NlpQueryState(
                     dialog.state.nextActionState?.states
                         ?: listOfNotNull(dialog.currentStory?.definition?.mainIntent()?.name).toSet()
-                )
+                ),
+                configuration =  dialog.playerIds.find { PlayerType.bot.equals(it.type) }?.id
```

Comment:
PlayerType.bot == it.type is better

Reply:
the bot player id is not always the configuration id. Please use `configuration = connector.botConfiguration.applicationId`